### PR TITLE
Update TablePicker.java

### PR DESCRIPTION
**Pull request by fpeter8**
Changed the table name format from 'YYYY' to 'yyyy' (Y is week year, y is year).

### CODE REVIEW

**Comment by fpeter8** [comment]
@ttozser

File: src/main/java/com/googlecode/objectify/insight/puller/TablePicker.java [hunk 1]
```diff
@@ -37,7 +37,7 @@
 	 * desired table name prefix.
 	 */
 	@Getter @Setter
-	private DateFormat format = new SimpleDateFormat("'OBJSTATS_'YYYYMMdd");;
```
Comment (nbali) [inline src/main/java/com/googlecode/objectify/insight/puller/TablePicker.java]: @fpeter8 modify the javadoc as well

Comment (fpeter8) [inline src/main/java/com/googlecode/objectify/insight/puller/TablePicker.java]: done

**Comment by nbali** [comment]
otherwise it's fine

**Pull request merged**